chromium_src/chrome/browser/profiles/pref_service_builder_utils.cc

```diff
@@ -40,13 +40,6 @@ void RegisterProfilePrefs(bool is_signin_profile,
   // Disable spell check service
   registry->SetDefaultPrefValue(
       spellcheck::prefs::kSpellCheckUseSpellingService, base::Value(false));
-
-  // Make sure sign into Brave is not enabled
-  // The older kSigninAllowed is deprecated and only in use in Android until
-  // C71.
-  registry->SetDefaultPrefValue(prefs::kSigninAllowedOnNextStartup,
```

Comment:
did you check if this will cause some part of UI on android showing up?
On desktop we need to do extra hiding for sync-accounts-control because of this pref

Reply:
Checking! thanks for the tip @darkdh